libbeat/publisher/queue/spool/outbroker.go

```diff
@@ -81,6 +82,12 @@ var ackChanPool = sync.Pool{
 var errRetry = errors.New("retry")
 
 func newOutBroker(ctx *spoolCtx, qu *pq.Queue, flushTimeout time.Duration) (*outBroker, error) {
+	reader := qu.Reader()
+	avail, err := reader.Available()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Should we log it?

Reply:
This error will be logged by the beat. The spool constructor fails if this call fails.